example/README.md

```diff
@@ -1,9 +1,12 @@
 # Example Setup
 
 0. Prerequisites:
-    - Python 3
-    - Docker and `docker-compose`
+    - Python 3.8+
+    - docker compose V2
     - ~16GiB of free storage
-2. Launch the containers: `docker-compose up -d`
+1. Clear local data dir if exists: `rm -vr data`
```

Comment:
I don't think this is neccessary and will cause unneccessary traffic.

Reply:
Your absolutely right! I just noticed this can do real harm. If the directory has been deleted and the containers create them from scratch it is going ill.

Reply:
Concerning `if only one single book missing ...` : The current flow is like this: one book is missing, then tar requests starts again. I tried the example in a place with slow net speed, lots of connection errs and required several tries. 
Before altering this I wasn't able to get just the books in place.